<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>V4LSettings</class>
 <widget class="QMainWindow" name="V4LSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="base">
     <attribute name="title">
      <string>&amp;Base Controls</string>
     </attribute>
     <widget class="QGroupBox" name="deviceInfoBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>460</width>
        <height>140</height>
       </rect>
      </property>
      <property name="title">
       <string>Device Info</string>
      </property>
      <widget class="QGroupBox" name="deviceBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>420</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>Device</string>
       </property>
       <widget class="QComboBox" name="device">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>380</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="imageSettingsBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>460</width>
        <height>140</height>
       </rect>
      </property>
      <property name="title">
       <string>Image Format Settings</string>
      </property>
      <widget class="QGroupBox" name="pixelSettingsBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>240</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>Pixel Format</string>
       </property>
       <widget class="QComboBox" name="pixelformat">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>200</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="frameSizeBox">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>Frame Size</string>
       </property>
       <widget class="QComboBox" name="frameSize">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>120</width>
          <height>24</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="advanced">
     <attribute name="title">
      <string>&amp;Advanced Controls</string>
     </attribute>
     <widget class="QGroupBox" name="advancedInfoBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>460</width>
        <height>140</height>
       </rect>
      </property>
      <property name="title">
       <string>Advanced Device Configuration</string>
      </property>
      <widget class="QGroupBox" name="captureModeBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>420</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>Device</string>
       </property>
       <widget class="QComboBox" name="captureMode">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>380</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="videoSettingsBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>460</width>
        <height>140</height>
       </rect>
      </property>
      <property name="title">
       <string>Video Format Settings</string>
      </property>
      <widget class="QGroupBox" name="videoModeBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>240</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>Video Mode</string>
       </property>
       <widget class="QComboBox" name="videoMode">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>200</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="colorSpaceBox">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>Color Space</string>
       </property>
       <widget class="QComboBox" name="colorSpace">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>120</width>
          <height>24</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="debug">
     <attribute name="title">
      <string>Debug &amp;Info</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>460</width>
        <height>440</height>
       </rect>
      </property>
      <property name="title">
       <string>Driver Capabilities</string>
      </property>
      <widget class="QListView" name="capabilities">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>420</width>
         <height>380</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="movement">
        <enum>QListView::Free</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>162</width>
      <height>25</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>V4LSettings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
